ci/cloudformation/github-actions-user.yml

```diff
@@ -0,0 +1,45 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: auth0-custom-lock GitHub Actions IAM User used to upload the NLX files to S3
+Metadata:
+  Source: https://github.com/mozilla-iam/auth0-custom-lock/tree/master/ci/cloudformation
+Resources:
+  GitHubActionsAuth0CustomLockUser:
+    Type: AWS::IAM::User
+    Properties:
+      Policies:
+        - PolicyName: AllowPutAuth0CustomLockFilesInS3
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
```

Comment:
Is this action (s3:ListAllMyBuckets) really needed?

Reply:
Not technically, but it’s a good safe operation that you can use to test things. @gene1wood says he usually adds it for that reason.

Reply:
:+1: